src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -312,6 +336,23 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
     }
   }
 
+  getDisableMigrate() {
+    if (
+      this.realms.length === 0 &&
+      this.zonegroups.length === 1 &&
+      this.zonegroups[0].name === 'default' &&
+      this.zones.length === 1 &&
+      this.zones[0].name === 'default'
```

Comment:
The button to migrate gets disabled if the name of the zonegroup and zone is not `default`? 
Why is that?

Reply:
You need a realm to enable the Migrate button, screenshot shows you do not have one.

Reply:
Why do we need a realm to enable a migrate button. If we have Zone group and a Zone/s at that time we need a migrate button and once we have a Realm -> Zonegroup and Zones available available we can assume we have migrated.

Reply:
Ohh! I mis-understood this to Import/export.! @cloudbehl  Yes you are correct. To answer @Pegonzal question - Yes, as per our implementation, we are allowing the user to Migrate from the very initial deployment that only has `default` zonegroup and zone.

Reply:
Then, @aaSharma14, shouldn't we be checking if the `zone` and `zonegroup` has the `default` property? Currently, we are checking  if their name is `default`, right? I'm not undersatanding this